tests/e2e/state.go

```diff
@@ -184,6 +185,8 @@ func (tr TestRun) getChainState(chain chainID, modelState ChainState) ChainState
 		chainState.RegisteredConsumerRewardDenoms = &registeredConsumerRewardDenoms
 	}
 
+	log.Println("Done getting chain state: ")
```

Comment:
nit: do we expect some variable after `:` in this print?

Reply:
Yes, thanks!